launcher/VersionProxyModel.cpp

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: GPL-3.0-only
 /*
  *  PolyMC - Minecraft Launcher
+ *  Prism Launcher - Minecraft Launcher
```

Comment:
this line should replace the PolyMC one, as it just shows which project this code currently belongs to.